src/status_im2/contexts/shell/jump_to/components/home_stack/view.cljs

```diff
@@ -27,7 +27,7 @@
              :z-index (get shared-values
                            (get shell.constants/stacks-z-index-keywords stack-id))})}
    (case stack-id
-     :communities-stack [communities/home]
+     :communities-stack [:f> communities/home]
```

Comment:
`communities/home` now need to be animated, it means need to use the `use-shared-value` hook, so it must be a functional component.